.github/workflows/test-website.yml

```diff
@@ -2,22 +2,19 @@ name: Test Website
 on:
   pull_request:
     branches: ["*"]
-    types: ["opened", "reopened"]
```

Comment:
I removed this because the linter was only running on the initial open. If you added a second commit, this workflow would not run, only the build workflow would run and all tests would show green. It was very misleading.